### ISSUE TITLE
0x691b692d9b181df61aba69f22f4721512012bd8a

### ISSUE THREAD

**Issue by qzz0518** [safe-global/safe-user-allocation-reports#91]
## Related Safe Addresses
0x5b953dfd01bf281974b296191fcbaa25ed6dd05f
0x7a901acf18a34ca90e329e7d9ac3141bbea45c9d
0xf7204998ae70da0d2df7cb41d490393e03c5133d
0x850f657376fd78e8fb581e9f1d4ac560f4094603
0xbefb0078f5e8ec9685de4ffa14f3605ef44d2e6b
0x805298584b55292215ac90391550641125a75d65
0x4e424f9a20c18cfe090f01262a2f239f32485728
0x6024e9b974f6643a8a9595b521e9d14feab77f0f
0x7cbc951103a0d880079deb94e98721d79e168757
0xea8206e4372bb589395502c8504ee0d39e313a2d

## Reasoning
These safe accounts are all created by 0x691b692d9b181df61aba69f22f4721512012bd8a
All 10 accounts carry out similar transactions, and all transactions only involve USDT and USDC, which are recharged at about the same time, and all token transfers are completed in about the same time by multiple transactions, and the balance is all 0. Withdrawals to the same address were found in multiple different accounts ，like ilovebourbon.eth

https://etherscan.io/tx/0x0a7ddb7701b9e00f537344675cf13db9f214569abbc2805734f137b981b709f9
https://etherscan.io/tx/0xbbedf32fbcc352c1ea7ce4f99b60e23b0ccd65681532f3ff4b9dac34c80b5ae4
https://etherscan.io/tx/0xc7234de17e198de1611ebb42dd425fdcd68d4540eece0006f14a487130e82e3e

## Methodology

Luabase was used for preliminary data removal, and then all TX were checked one by one by EtherScan

## My Safe Address
0xf4ed1EF1f7A52DC7Dd10a9B5d43f23748A1E8992

**Comment by tschubotz** [comment]
> These safe accounts are all created by 0x691b692d9b181df61aba69f22f4721512012bd8a
> All 10 accounts carry out similar transactions, and all transactions only involve USDT and USDC, which are recharged at about the same time, and all token transfers are completed in about the same time by multiple transactions, and the balance is all 0. Withdrawals to the same address were found in multiple different accounts ，like ilovebourbon.eth

This report doesn't meet the quality requirement. It's hard to following. Please elaborate and provide structure to your reasoning, incl. clarifying the following:

> All 10 accounts carry out similar transactions,

what txs?

> which are recharged at about the same time

what means "recharging" and why is this airdrop farming?

> Withdrawals to the same address were found in multiple different accounts ，like ilovebourbon.eth

Pls provide the complete picture and why this is airdrop farming.

**Comment by lukasschor** [comment]
We haven't received any further information on this ticket and therefore deemed this submission invalid.

Unfortunately just the pure fact that Safes...

- were created by the same address
- have the same owner
- conducted the same transactions
- received assets from the same address

... is not enough reasoning for a valid submission, as there are legit user behaviour leading to such as well (relayer, scripts, power users, exchanges)

This submission didn't provide sufficient indicators to convince the team that there is some targeted airdrop farming behind the provided addresses. As per the rules, when in doubt we will default in favor of the airdrop recipients in order to not risk any false positives when sorting out airdrop hunters.
